### PR TITLE
Login gets an access_token

### PR DESCRIPTION
**Pull request by cbas**
Depends on migme/beachball#40

### CODE REVIEW

File: app/components/mig/menu/menu.styl [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+  margin 30px 0
+
+  svg
+    midth 150px
+    max-width 50%
+
+nav
+  text-align center
+
+  button
```
Comment (eddiemoore) [inline app/components/mig/menu/menu.styl]: If it's a web component, you shouldn't need to embed ```button``` in ```nav```
```css
nav button {}
```
could just be
```css
button {}
```

Comment (cbas) [inline app/components/mig/menu/menu.styl]: Because multiple buttons are grouped and I wanted a nice way to put them in
a block element. Otherwise they kind of float at the top level of the
domfragment.

Ok flying noooooow...

File: app/components/mig/login/login.styl [hunk 2]
```diff
@@ -1,3 +1,50 @@
+$mantis = #3cb9a0
+$carrotOrange = #ec932b
```
Comment (eddiemoore) [inline app/components/mig/login/login.styl]: You are using this colour in **buttons.styl**. Might want to put the colour variables into their own file and import them into both.

File: app/components/mig/login/wb_setting.svg [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (eddiemoore) [inline app/components/mig/login/wb_setting.svg]: SVG cleanup?

File: app/components/mig/menu/menu.styl [hunk 4]
```diff
@@ -1,2 +1,30 @@
-button
-  border 10px solid gold
+$mantis = #3cb9a0
```
Comment (eddiemoore) [inline app/components/mig/menu/menu.styl]: Put into a colour variables file

File: app/components/mig/menu/wb_migme_explore_colour.svg [hunk 5]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (eddiemoore) [inline app/components/mig/menu/wb_migme_explore_colour.svg]: :trollface:

File: app/components/mig/panel/panel.styl [hunk 6]
```diff
@@ -1,6 +1,58 @@
+$mantis = #3cb9a0
```
Comment (eddiemoore) [inline app/components/mig/panel/panel.styl]: :trollface:

File: app/components/mig/panel/wb_chatclose.svg [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (eddiemoore) [inline app/components/mig/panel/wb_chatclose.svg]: :trollface:

File: app/components/mig/profile/profile.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+class Profile extends HTMLElement {
+  createdCallback () {
+    this.createShadowRoot()
+    this.shadowRoot.innerHTML = html()
+    insertCss(css, { parent: this.shadowRoot })
+  }
+  attachedCallback () {
+    this::closest('mig-me').beachball.API.url('me')
+      .then(profile => {
+        // console.log({profile})
```
Comment (eddiemoore) [inline app/components/mig/profile/profile.js]: :trollface:

File: app/components/mig/login/wb_setting.svg (hunk 3 above)
Comment (cbas) [inline app/components/mig/login/wb_setting.svg]: These were the raw assets from Google Drive. If this is important you might want to look at using SVGO. Seems outside the scope of this PR though.

File: app/components/mig/login/login.styl (hunk 2 above)
Comment (cbas) [inline app/components/mig/login/login.styl]: Would break encapsulation of web components to have the `.styl` file import from a parent directory. I would instead look at this approach: https://github.com/DamonOehlman/stylify#adding-paths-for-require

**Comment by eddiemoore** [comment]
:ship: :shipit: :ship:

**Comment by cbas** [comment]
:bow:
